src/autocomplete/index.jsx

```diff
@@ -19,6 +19,9 @@ export default function AutoComplete(props) {
       if (!val) {
         setValue('');
       }
+      if (props.stringValue) {
```

Comment:
feel free to suggest a better prop name for when options is a list of strings, not value/label pairs